tests/distro/test_default.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2014 Microsoft Corporation
```

Comment:
for clarity yes it should, though it would be found by the test runner either way; moved it over.